src/promptflow/promptflow/_sdk/_serving/app.py

```diff
@@ -32,7 +31,8 @@
 
 from .swagger import generate_swagger
 
-logger = LoggerFactory.get_logger(LOGGER_NAME)
+logger = LoggerFactory.get_logger("pfserving-app", target_stdout=True)
+print("second")
```

Comment:
why this?

Reply:
To avoid initialization conflict with other logger in sdk,  and given the log all comes from promptflow serving app,  so using this name.